src/ObjectUtils/CMakeLists.txt

```diff
@@ -31,7 +33,9 @@ target_sources(
         WindowsBuildIdUtils.cpp)
 
 if (NOT WIN32)
-target_sources(ObjectUtils PRIVATE LinuxMap.cpp)
+target_sources(ObjectUtils PRIVATE
+        ReadMaps.cpp
```

Comment:
Didn't we agreed at some point that platform specific files end with the platform? I.e. ReadMapsLinux.h/.cpp?

Reply:
Hmm, that was a bit of a different case. Basically if you have a header that is platform-agnostic but platform-specific implementation files for the platform-specific implementations, you have `MyUtils.h` but `MyUtilsLinux.cpp` and `MyUtilsWindows.cpp`. But this is all Linux-only, it gets very repetitive to have a "Linux" suffix for every Linux-only header. Other platform specific files in this module also don't have a suffix.

`ReadModules` should probably be platform-agnostic, yeah... but the Windows version is in a different module altogether, so that's a story for another day.